CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Change log
 
+## 0.12.0
+
+### Changed
+
+- Using bref/build-php-XX:1.2.12 as base image
```

Comment:
Will be 1.2.13 after merge of https://github.com/brefphp/bref/pull/1008 which is an upfix